docs/recipes/index.txt

```diff
@@ -0,0 +1,99 @@
+.. _recipes:
```

Comment:
Perhaps move this to a separate PR? It seems mostly independent of the other change.